app/assets/javascripts/subscription.js

```diff
@@ -0,0 +1,59 @@
+$(document).ready(function(){
+  $("#btn_subscribe").click(function (){
+    $("#subscribe").closeModal();
+    var event_id, data;
```

Comment:
I implemented it initially like this, but @andela-sdamian recommended using single vars. I will change it immediately.